Formula/wavpack.rb

```diff
@@ -11,27 +12,18 @@ class Wavpack < Formula
     sha256 "8edf6d12295eb29d88e38d5c9618504508db63367ceea9f16a7e3aa4d9cfeebc" => :yosemite
   end
 
-  head do
-    url "https://github.com/dbry/WavPack.git"
-    depends_on "automake" => :build
-    depends_on "autoconf" => :build
-    depends_on "libtool" => :build
-  end
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
 
   def install
-    args = %W[--prefix=#{prefix} --disable-dependency-tracking]
-
-    if build.head?
-      system "./autogen.sh", *args
-    else
-      system "./configure", *args
-    end
-
+    system "./autogen.sh", "--prefix=#{prefix}",
```

Comment:
if you add the argument `--enable-tests`, you can compile the full test suite for testings